spec/unit/validator_spec.rb

```diff
@@ -600,6 +600,39 @@ def podspec_path(name = 'JSONKit', version = '1.4')
         validator.results_message.strip.should.be.empty
       end
 
+      it 'uses the deployment target of the current test spec' do
```

Comment:
it's not super clear to me what in here is validating that the correct deployment target is being used for xcodebuild ?

Reply:
it pulls a destination from `Fourflusher` once requesting iOS 8 for building the library and once requesting iOS 9 to run the test. at least that was my understanding. maybe would be better to add an expectation that `xcodebuild` is provided the correct deployment targets instead

Reply:
yeap. Those tests are super convoluted and hard to follow. I verified without my change this test fails.